app/api/get-apim-headers.js

```diff
@@ -0,0 +1,26 @@
+const { apimConfig } = require('../config')
+
+const getApimHeaders = (headers, apimToken) => {
+  const apimHeaders = {
+    'Ocp-Apim-Subscription-Key': apimConfig.ocpSubscriptionKey,
+    Authorization: apimToken
+  }
+
+  if (headers.crn) {
+    apimHeaders.crn = headers.crn
+  }
+
+  if (headers.email) {
+    apimHeaders.email = headers.email
+  }
+
+  if (headers.authorization) {
+    apimHeaders['X-Forwarded-Authorization'] = headers.authorization
+  }
```

Comment:
APIM request now only includes these headers if they were included on the original request